pkg/imagedistributor/mount.go

```diff
@@ -50,7 +54,13 @@ func (b buildAhMounter) Mount(imageName string, platform v1.Platform) (string, e
 		return "", err
 	}
 
-	if _, err := b.imageEngine.CreateWorkingContainer(&imagecommon.BuildRootfsOptions{
+	// make sure base mount Dir is existed.
+	err := fs.FS.MkdirAll(filepath.Dir(mountDir))
+	if err != nil {
```

Comment:
consider the following syntax to avoid widening the scope of error

if err := fs.FS.MkdirAll(filepath.Dir(mountDir));err!=nil{}